### PR TITLE
Upgrade wp-prettier to 2.2.1

### PR DESCRIPTION
**Pull request by jsnajdr**
Upgrades `wp-prettier` to version 2.2.1 of the fork, making it up to date with the latest upstream.

I already upgraded Calypso successfully earlier today: https://github.com/Automattic/wp-calypso/pull/47950

### CODE REVIEW

File: packages/data/src/types.d.ts [hunk 1]
```diff
@@ -2,31 +2,31 @@ export type WPDataFunctionOrGeneratorArray = Array< Function | Generator >;
 export type WPDataFunctionArray = Array< Function >;
 
 export interface WPDataAttachedStore {
```
Comment (gziolo) [inline packages/data/src/types.d.ts:4]: Is it using now different indentation than JS? Can we align?

**Review by gziolo — COMMENTED** [review]
I had one comment related to TS files but otherwise it's looking good. I'm never sure if this is only enhancement knowing that we also use Prettier for linting.

**Comment by gziolo** [comment]
Hmm, I can't edit my comment on GitHub mobile 🙃 I meant that it would be great to verify if it isn't a breaking change because of linting integration.

File: packages/data/src/types.d.ts (hunk 1 above)
Comment (ntwb) [reply packages/data/src/types.d.ts:4]: This looks correct to me @gziolo, the indentation is changes spaces to tabs, I suspect this had been missed by ESLint/Prettier at some point previously:

image

Comment (jsnajdr) [reply packages/data/src/types.d.ts:4]: Yes, this file previously used spaced and was merely reformatted to tabs. There's been no radical change to how TS files are formatted and there are no differences between JS an TS formatting.

Comment (jsnajdr) [reply packages/data/src/types.d.ts:4]: Now I noticed that `tsc` uses 4 spaces indentation when generating the `*.d.ts` files in `build-types`. That's probably where this formatting originated. And Babel apparently indents with 2 spaces.

Comment (gziolo) [reply packages/data/src/types.d.ts:4]: I was browsing this file on mobile, I could verify whether it's tabs or spaces, it looked like 2 spaces. Ignore this comment One tab is correct.

**Review by gziolo — APPROVED** [review]
Cool, thank you @jsnajdr.

**Pull request merged**